WordPress/src/main/java/org/wordpress/android/ui/reader/subfilter/SubFilterViewModel.kt

```diff
@@ -124,7 +139,12 @@ class SubFilterViewModel @Inject constructor(
                             blog.organizationId == organization.orgId
                         } ?: false
                     }
-                }
+                }.sortedWith(Comparator { blog1, blog2 ->
+                    // sort followed blogs by name/domain to match display
+                    val blogOneName = getBlogNameForComparison(blog1)
```

Comment:
Nitpick: would it make sense to have the same standard for the variable name with `blogOneName` and `blogNameTwo`? 

e.g.
`blogOneName` and `blogTwoName` 

or

`blogNameOne` and `blogNameTwo`

Reply:
Yes, right. Sorry about that.